zebra-network/src/peer_set/initialize.rs

```diff
@@ -228,36 +214,71 @@ where
     Ok(())
 }
 
-/// Listens for peer connections on `addr`, then sets up each connection as a
-/// Zcash peer.
+/// Open a peer connection listener on `config.listen_addr`,
+/// returning the opened `TcpListener`, and the address it is bound to.
```

Comment:
Nit: adding a link to Tokio's documentation.

```suggestion
/// returning the opened [`TcpListener`], and the address it is bound to.
```